app/pods/publications/publication/controller.js

```diff
@@ -13,7 +13,7 @@ export default class PublicationController extends Controller {
   @service toaster;
   @service intl;
   @service media;
-
+  @tracked publicationNotAfterTranslation = false;
```

Comment:
Wordt deze state deftig reset tussen modellen, ga dit manueel testen.
Anders moet die in een resetController